LEAF_Nexus/sources/Employee.php

```diff
@@ -75,6 +77,460 @@ public static function cleanWildcards($input)
+     * @param array $national_list
+     * @param array $local_list
+     *
+     * @return void
+     *
+     * Created at: 6/9/2023, 2:30:08 PM (America/New_York)
+     */
+    private function prepareArrays(array &$national_employee_uids, array &$local_employee_array, array $national_list, array $local_list): void
+    {
+        foreach ($national_list as $employee) {
+            $national_employee_uids[] = (int) $employee['empUID'];
+
+            $local_employee_array[] = [
+                'empUID' => (empty($local_list[$employee['userName']]) ? null : $local_list[$employee['userName']]),
+                'userName' => $employee['userName'],
+                'lastName' => $employee['lastName'],
+                'firstName' => $employee['firstName'],
+                'middleName' => $employee['middleName'],
+                'phoneticFirstName' => $employee['phoneticFirstName'],
+                'phoneticLastName' => $employee['phoneticLastName'],
+                'domain' => $employee['domain'],
+                'deleted' => $employee['deleted'],
+                'lastUpdated' => $employee['lastUpdated']
+            ];
+        }
+    }
+
+    /**
+     * @param array $local_data_array
+     * @param array $national_list
+     * @param array $local_list
+     *
+     * @return void
+     *
+     * Created at: 6/9/2023, 2:30:17 PM (America/New_York)
+     */
+    private function prepareDataArray(array &$local_data_array, array $national_list, array $local_list): void
+    {
+        foreach ($national_list as $employee) {
+            $local_data_array[] = [
+                'empUID' => (empty($local_list[$employee['userName']]) ? null : $local_list[$employee['userName']]),
+                'indicatorID' => $employee['indicatorID'],
+                'data' => $employee['data'],
+                'author' => $employee['author'],
+                'timestamp' => $employee['timestamp'],
+            ];
+        }
+    }
+
+    /**
+     * @param array $employee_list
+     *
+     * @return array
+     *
+     * Created at: 6/9/2023, 2:30:29 PM (America/New_York)
+     */
+    private function userNameUidList(array $employee_list): array
+    {
+        $return_value = [];
+
+        foreach ($employee_list as $employee) {
+            $return_value[$employee['userName']] = $employee['empUID'];
+        }
+
+        return $return_value;
+    }
+
+    /**
+     * @param Db $db
+     *
+     * @return array
+     *
+     * Created at: 6/9/2023, 2:30:36 PM (America/New_York)
+     */
+    private function getAllEmployees(Db $db): array
+    {
+        $vars = array();
+        $sql = 'SELECT userName
+                FROM employee';
+
+        $return_value = $db->prepared_query($sql, $vars);
+
+        return $return_value;
+    }
+
+    /**
+     * @param array $user
+     *
+     * @return bool
+     *
+     * Created at: 6/9/2023, 2:30:52 PM (America/New_York)
+     */
+    private function isActiveNationally(array $user): bool
```

Comment:
'Course if all you're returning is a boolean, not sure there is a reason to return anything else.